docs/reference/commandline/container_run.md

```diff
@@ -1363,6 +1363,7 @@ in the image, or `SIGTERM` if the image has no `STOPSIGNAL` defined.
 | `--security-opt="seccomp=unconfined"`     | Turn off seccomp confinement for the container                                                                                                                                                                   |
 | `--security-opt="seccomp=builtin"`        | Use the default (built-in) seccomp profile for the container. This can be used to enable seccomp for a container running on a daemon with a custom default profile set, or with seccomp disabled ("unconfined"). |
 | `--security-opt="seccomp=profile.json"`   | White-listed syscalls seccomp Json file to be used as a seccomp filter                                                                                                                                           |
+| `--security-opt="systempaths=unconfined"` | Turn off confinment for system paths (masked paths, read-only paths) for the container                                                                                                                           |
```

Comment:
nit: typo in `confinement`

Reply:
Aaaah, thank you. Too many squigglies going on here 😆 
<img width="584" alt="image" src="https://github.com/docker/cli/assets/35727626/3124e5f5-b904-4102-a708-a548ab9c3448">